authui/src/modal.ts

```diff
@@ -1,89 +1,66 @@
+import { Controller } from "@hotwired/stimulus";
+
 // usage: adding follow data attribute in the button element
-// - data-modal="confirmation"
+// - data-modal-target="submitButton"
```

Comment:
It is better not to store this target. Instead, we should store the e.currentTarget in `confirmFormSubmit`. When confirm, call the stored target click(). This approach allows us to support popping up different modal in the same page.